views/playground/workflow-designer/index.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends '@pulsar/pulsar/layouts/base.html.twig' %}
```

Comment:
Probably not an issue with it being a playground but there are some validation and a11y tools errors present on this page.